src/NzbDrone.Core/Applications/Lidarr/Lidarr.cs

```diff
@@ -95,6 +95,14 @@ public override void AddIndexer(IndexerDefinition indexer)
             var lidarrIndexer = BuildLidarrIndexer(indexer, indexer.Protocol);
 
             var remoteIndexer = _lidarrV1Proxy.AddIndexer(lidarrIndexer, Settings);
+
+            if (remoteIndexer == null)
+            {
+                _logger.Warn("Remote indexer not found, failed to add {0} [{1}].", indexer.Name, indexer.Id);
```

Comment:
The Proxy already logs an error it seems, do we really need the second one?

![image](https://github.com/Prowlarr/Prowlarr/assets/376117/95a6957a-125f-4bd1-95a7-a863f3485ef2)

Reply:
https://github.com/Prowlarr/Prowlarr/commit/65d6d518d78e45dcd9257536a93fe9ef4e0d069b seems to be the cause of this. Looks like that commit has errors pretty covered